vulnerability_id: VCID-r5ua-w32z-aaag
aliases:
  - CVE-2014-3589
  - PYSEC-2014-10
summary: PIL/IcnsImagePlugin.py in Python Imaging Library (PIL) and Pillow before 2.3.2 and
  2.5.x before 2.5.2 allows remote attackers to cause a denial of service via a crafted block
  size.
severities:
  - id: 656468
    reference_id: 1424922
    scoring_system: generic_textual
    value: Low
    scoring_elements:
  - id: 656469
    reference_id: 1424923
    scoring_system: generic_textual
    value: Low
    scoring_elements:
  - id: 656470
    reference_id: 1424924
    scoring_system: generic_textual
    value: Medium
    scoring_elements:
  - id: 58837
    reference_id: 185259
    scoring_system: rhbs
    value: medium
    scoring_elements:
  - id: 180469
    reference_id: 566844
    scoring_system: cvssv2
    value: '5.0'
    scoring_elements: AV:N/AC:L/Au:N/C:N/I:N/A:P
references:
  - id: 4430
    url: http://lists.opensuse.org/opensuse-updates/2015-04/msg00056.html
    reference_id:
  - id: 8168
    url: http://osvdb.org/show/osvdb/110128
    reference_id:
  - id: 1424922
    url: http://people.canonical.com/~ubuntu-security/cve/2014/CVE-2014-3589.html
    reference_id:
  - id: 185260
    url: https://access.redhat.com/hydra/rest/securitydata/cve/CVE-2014-3589.json
    reference_id:
  - id: 1424923
    url: https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2014-3589
    reference_id:
  - id: 8169
    url: http://seclists.org/bugtraq/2014/Sep/25
    reference_id:
  - id: 4431
    url: http://secunia.com/advisories/59825
    reference_id:
  - id: 4429
    url: https://github.com/python-pillow/Pillow/commit/205e056f8f9b06ed7b925cf8aa0874bc4aaf8a7d
    reference_id:
  - id: 4427
    url: https://pypi.python.org/pypi/Pillow/2.3.2
    reference_id:
  - id: 4426
    url: https://pypi.python.org/pypi/Pillow/2.5.2
    reference_id:
  - id: 1424924
    url: https://ubuntu.com/security/notices/USN-3080-1
    reference_id:
  - id: 4428
    url: http://www.debian.org/security/2014/dsa-3009
    reference_id:
  - id: 185259
    url: https://bugzilla.redhat.com/show_bug.cgi?id=1130711
    reference_id: 1130711
  - id: 100199
    url: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=758772
    reference_id: 758772
  - id: 566845
    url: https://nvd.nist.gov/vuln/search/results?adv_search=true&isCpeNameSearch=true&query=cpe:2.3:a:debian:python-imaging:-:*:*:*:*:*:*:*
    reference_id: cpe:2.3:a:debian:python-imaging:-:*:*:*:*:*:*:*
  - id: 558744
    url: https://nvd.nist.gov/vuln/search/results?adv_search=true&isCpeNameSearch=true&query=cpe:2.3:a:python:pillow:*:*:*:*:*:*:*:*
    reference_id: cpe:2.3:a:python:pillow:*:*:*:*:*:*:*:*
  - id: 563402
    url: https://nvd.nist.gov/vuln/search/results?adv_search=true&isCpeNameSearch=true&query=cpe:2.3:a:python:pillow:2.3.0:*:*:*:*:*:*:*
    reference_id: cpe:2.3:a:python:pillow:2.3.0:*:*:*:*:*:*:*
  - id: 566846
    url: https://nvd.nist.gov/vuln/search/results?adv_search=true&isCpeNameSearch=true&query=cpe:2.3:a:python:pillow:2.5.0:*:*:*:*:*:*:*
    reference_id: cpe:2.3:a:python:pillow:2.5.0:*:*:*:*:*:*:*
  - id: 566847
    url: https://nvd.nist.gov/vuln/search/results?adv_search=true&isCpeNameSearch=true&query=cpe:2.3:a:python:pillow:2.5.1:*:*:*:*:*:*:*
    reference_id: cpe:2.3:a:python:pillow:2.5.1:*:*:*:*:*:*:*
  - id: 566848
    url: https://nvd.nist.gov/vuln/search/results?adv_search=true&isCpeNameSearch=true&query=cpe:2.3:a:python:pillow:2.5.2:*:*:*:*:*:*:*
    reference_id: cpe:2.3:a:python:pillow:2.5.2:*:*:*:*:*:*:*
  - id: 157778
    url: https://nvd.nist.gov/vuln/search/results?adv_search=true&isCpeNameSearch=true&query=cpe:2.3:o:opensuse:opensuse:13.2:*:*:*:*:*:*:*
    reference_id: cpe:2.3:o:opensuse:opensuse:13.2:*:*:*:*:*:*:*
  - id: 566844
    url: https://nvd.nist.gov/vuln/detail/CVE-2014-3589
    reference_id: CVE-2014-3589
weaknesses: []
